napi-inl.h

```diff
@@ -425,6 +425,38 @@ inline std::u16string String::Utf16Value() const {
 // Object class
 ////////////////////////////////////////////////////////////////////////////////
 
+namespace helper {
+
+template <typename Key>
+class PropertyLValue {
```

Comment:
I’d prefer to keep a name that reflects that this is something that’s meant to be *assignable*. Calling it something along the lines of `PropertyReference` also crossed my mind (e.g. comparable to `std::bitset<N>::reference`), that works for me too.